src/Platforms/PostgreSQLPlatform.php

```diff
@@ -809,6 +809,18 @@ public function getDropForeignKeySQL($foreignKey, $table)
         return $this->getDropConstraintSQL($foreignKey, $table);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getDropIndexSQL($index, $table = null)
+    {
+        if ($index->isPrimary()) {
+            return $this->getDropConstraintSQL($table.'_pkey', $table);
```

Comment:
What if the constraint is not using the default name? https://www.postgresql.org/docs/current/sql-createtable.html#id-1.9.3.85.1.